tools/topology/topology1/sof-tgl-max98373-rt5682.m4

```diff
@@ -50,6 +50,8 @@ define(`SMART_SSP_NAME', concat(concat(`SSP', AMP_SSP),`-Codec'))
 define(`SMART_BE_ID', 7)
 #define SSP mclk
 define(`SSP_MCLK', 24576000)
+# Run Smart Amp pipeline on core#1 by default for low power considering
+define(`SMART_AMP_CORE', 1)
```

Comment:
is the addition of a separate variable from SPK_PLAYBACK_CORE intentional?

This could lead to a case where SMART_AMP_CORE=1 and SPK_PLAYBACK_CORE=0 - not sure if this is desired or that it even works since the two pipelines are connected.

Should this be 
````
define(`SMART_AMP_CORE', SPK_PLAYBACK_CORE)
````
??

Reply:
Sure. Thanks for the suggestion.